### PR TITLE
Steph dev

### PR DESCRIPTION
**Pull request by sgh005**
First pull request! Did the bare minimum and trying not to do everything at once in the end. As of now I just changed some of the title and added a picture (:

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,36 @@
...
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+        <center>
+      <h1>This is Me!</h1>
+      <p>I'm a 21 year old first gen student from LA going to UCSD
+         I'm passionate about outreach and social justice. I'm currently the 
+          outreach co-chair for Women in Computing and want to start a Latinx's in
```
Comment (alisonberkowitz) [inline portfolio/src/main/webapp/index.html]: love this!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,36 @@
...
+          tech mini org. I love computer scinece and problem solving and I also 
+          love/want to learn more about social activism and movements.
+      </p>
+      <p>My plans post graduation include going into industry and <i>ideally</i>
+      working for a company that does community focused work </p>
+        <p> In my free time I like to go down rabbit holes on random subjects,
+            make Mexican food vegan, eating brunch from many of La Jolla's brunch
+            spots, and watching movies and the LA Lakers
```
Comment (alisonberkowitz) [inline portfolio/src/main/webapp/index.html]: uh oh I happen to be a Celtics fan :stuck_out_tongue_closed_eyes: but some former Celtics are now on the Lakers so we can bond over that :smile:

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,36 @@
...
+        </p>
+        <p>Feel free to recommend readings on social justice and related topics!</p>
+
+      
+    <img src="images/31.png"/><p>This is a lil me I got at the Google 
+          space at the Grace Hopper Conference 2018!</p>
       <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <button onclick="randomXDQuote()">Random Quote!</button>
```
Comment (alisonberkowitz) [inline portfolio/src/main/webapp/index.html]: does this log an error when you click the button? I don't see a function named randomXDQuote defined anywhere. Feel free to add an empty function with a comment to implement it in a future PR (pull request) to your script.js file.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (sgh005) [reply portfolio/src/main/webapp/index.html]: Rondo really is making peace !!!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (sgh005) [reply portfolio/src/main/webapp/index.html]: I realized I didn't add script.js, so it's not updated here :0. I'll add right now!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (sgh005) [reply portfolio/src/main/webapp/index.html]: (:

**Pull request merged**